apps/bare-expo/ios/Podfile.lock

```diff
@@ -1,8 +1,8 @@
 PODS:
   - boost (1.76.0)
-  - CocoaLumberjack (3.7.4):
-    - CocoaLumberjack/Core (= 3.7.4)
-  - CocoaLumberjack/Core (3.7.4)
+  - CocoaLumberjack (3.8.0):
+    - CocoaLumberjack/Core (= 3.8.0)
+  - CocoaLumberjack/Core (3.8.0)
```

Comment:
Could you revert changes in the lock and rerun `pod install`? It's better not to upgrade unrelated dependencies 😉